src/main/java/com/raisetech10/liquor_management/controller/LiquorController.java

```diff
@@ -57,6 +57,14 @@ public ResponseEntity<LiquorResponse> updateLiquor(@PathVariable("id") int id, @
         return ResponseEntity.ok(new LiquorResponse("★a liquor is updated★"));
     }
 
+    //DELETE
+    //お酒のデータを削除
+    @DeleteMapping("/liquor/{id}") //idを指定して削除
```

Comment:
liquorは不可算名詞ではあるものの複数種のお酒を扱うという意味でliquorsにするのも違和感はない。
```suggestion
    @DeleteMapping("/liquors/{id}") //idを指定して削除
```

Reply:
複数形のsわかりました。わかりやすくする為に、使います。